recipes/docdata/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.0.4" %}
+# Latest python_min is defined in https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml
+
+package:
+  name: docdata
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/d/docdata/docdata-{{ version }}.tar.gz
+  sha256: 270ace9349e2a71c2bd38c83fd9437cd4cbd1c4fb765016138976f0898c4f457
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```

Comment:
You don't need this for noarch packages.

```suggestion
```

Reply:
True, but doesn't it result in 1 CI build rather than 3?

Reply:
Yes, and that is the main advantage of noarch. 1 build, install in all platforms and archs!